### PR TITLE
Add a short deployment guide for 64-bit OSX standalones

### PR DESCRIPTION
**Pull request by livecodefraser**

### CODE REVIEW

File: docs/guides/Mac Deployment.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Introduction
+
+LiveCode 8.0 DP15 introduced the ability to build 64-bit standalones for MacOSX after a successful sponsorship campaign on the LiveCode Feature Exchange.
```
Comment (peter-b) [inline docs/guides/Mac Deployment.md:3]: Let's just say "LiveCode 8" here.  Presumably, if people have got the ability to deploy to Mac OS 64-bit, then they must have a sufficiently recent DP!

File: docs/guides/Mac Deployment.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* Sound recording is not available as it depends on QuickTime
+* The revvideograbber external is not available as it depends on QuickTime
+
+Some other features may differ in minor ways not affecting functionality, for example:
+
+* Printing progress dialogs are not shown as the necessary API was removed by Apple
+
+# The IDE
+
+The LiveCode IDE runs in 32-bit mode by default for maximum compatibility. If you'd like to run it in 64-bit mode, right-click on the app bundle, select "Get Info" and deselect the "Open in 32-bit mode" option. In future, we will add a menu item in the IDE that re-launches the IDE in either 32-bit or 64-bit mode, as appropriate.
```
Comment (peter-b) [inline docs/guides/Mac Deployment.md:19]: Please make numbered steps.

**Comment by peter-b** [comment]
We need this for dp-15, and the issues are purely cosmetic.

@livecode-vulcan review ok 659c84c

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 659c84c1f1de5ede9c048061cfb6344442293a66

**Comment by livecode-vulcan** [comment]
😎 test success 659c84c1f1de5ede9c048061cfb6344442293a66
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1839)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1852)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1833)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1845)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1855)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1858)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1864)

**Pull request merged**